controllers/comments.controllers.js

```diff
@@ -0,0 +1,20 @@
+const { selectCommentsByArticleId } = require("../models/comments.models");
+const { checkExists } = require("../db/helpers/utils");
+
+exports.getCommentsByArticleId = (req, res, next) => {
+  const articleId = req.params.article_id;
```

Comment:
This could also be done with destructuring - up to you :) i.e. `const {article_id) = req.params`